core/src/main/java/kafka/docker/Log4jConfiguration.java

```diff
@@ -0,0 +1,221 @@
+
+    @JsonProperty("Properties")
+    public Properties getProperties() {
+        return properties;
+    }
+
+    public void setProperties(Properties properties) {
+        this.properties = properties;
+    }
+
+    @JsonProperty("Appenders")
+    public Appenders getAppenders() {
+        return appenders;
+    }
+
+    public void setAppenders(Appenders appenders) {
+        this.appenders = appenders;
+    }
+
+    @JsonProperty("Loggers")
+    public Loggers getLoggers() {
+        return loggers;
+    }
+
+    public void setLoggers(Loggers loggers) {
+        this.loggers = loggers;
+    }
+
+    @JsonAnyGetter
+    public Map<String, Object> getAdditionalProperties() {
+        return additionalProperties;
+    }
+
+    @JsonAnySetter
+    public void setAdditionalProperties(String key, Object value) {
+        additionalProperties.put(key, value);
+    }
+}
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+class Properties {
+    private final Map<String, Object> properties = new LinkedHashMap<>();
+
+    @JsonAnyGetter
+    public Map<String, Object> getProperties() {
+        return properties;
+    }
+
+    @JsonAnySetter
+    public void setProperties(String key, Object value) {
+        properties.put(key, value);
+    }
+}
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+class Appenders {
+    private final Map<String, Object> properties = new LinkedHashMap<>();
+
+    @JsonAnyGetter
+    public Map<String, Object> getProperties() {
+        return properties;
+    }
+
+    @JsonAnySetter
+    public void setProperties(String key, Object value) {
+        properties.put(key, value);
+    }
+}
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+class Loggers {
+    private Root root;
+    private List<Logger> logger = Collections.emptyList();
+
+    @JsonProperty("Root")
+    public Root getRoot() {
+        return root;
+    }
+
+    public void setRoot(Root root) {
+        this.root = root;
+    }
+
+    @JsonProperty("Logger")
+    public List<Logger> getLogger() {
+        return logger;
+    }
+
+    public void setLogger(List<Logger> logger) {
+        this.logger = logger;
+    }
+}
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+class Root {
+    private String level;
+    private final Map<String, Object> otherProperties = new LinkedHashMap<>();
+
+    public String getLevel() {
+        return level;
+    }
+
+    public void setLevel(String level) {
+        this.level = level;
+    }
+
+    @JsonAnyGetter
+    public Map<String, Object> getOtherProperties() {
+        return otherProperties;
+    }
+
+    @JsonAnySetter
+    public void setOtherProperties(String key, Object value) {
+        otherProperties.put(key, value);
+    }
+}
+
+@JsonPropertyOrder({ "name", "level" })
+@JsonIgnoreProperties(ignoreUnknown = true)
+class Logger {
+    private String name;
+    private String level;
+    private final Map<String, Object> otherProperties = new LinkedHashMap<>();
```

Comment:
`KAFKA_LOG4J_LOGGERS` is only used for setting logger names and levels, so we don't need to consider other properties here. 
This property is for serialization purposes.